webapp/scss.md

```diff
@@ -0,0 +1,73 @@
+# SCSS - Code Guidelines
+
+## We use classes over tags and ids
+
+Never apply styles to `tags` or `ids` – use `classes` instead!
+
+Why?
+- HTML tags are responsible for the document _structure_, not the looks
+- targeting HTML tags comes with performance issues
+- ids are responsible for identifying a unique element, not for styling it
+- ids have higher specificity than classes and therefore don't play well together
+- classes can be combined and reused while ids are unique
+
+For more background see the following articles on [why not to style tags](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css) and [why not to style ids](https://dev.to/clairecodes/reasons-not-to-use-ids-in-css-4ni4).
+
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-10-31T10:44:06Z" title="Thursday, October 31st 2019, 11:44:06 am +01:00">Oct 31, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,73 @@
+# SCSS - Code Guidelines
+
+## We use classes over tags and ids
+
+Never apply styles to `tags` or `ids` – use `classes` instead!
+
+Why?
+- HTML tags are responsible for the document _structure_, not the looks
+- targeting HTML tags comes with performance issues
+- ids are responsible for identifying a unique element, not for styling it
+- ids have higher specificity than classes and therefore don't play well together
+- classes can be combined and reused while ids are unique
+
+For more background see the following articles on [why not to style tags](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css) and [why not to style ids](https://dev.to/clairecodes/reasons-not-to-use-ids-in-css-4ni4).
+
```


I feel smarter for having read these guidelines!!